content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -197,9 +166,9 @@ Then use the functions and data structures in the stub file to develop the serve
 
 ##### KMS v2 {#developing-a-kms-plugin-gRPC-server-notes-kms-v2}
 
-* KMS plugin version: `v2beta1`
+* KMS plugin version: `v2`
 
-  In response to procedure call `Status`, a compatible KMS plugin should return `v2beta1` as `StatusResponse.version`,
+  In response to procedure call `Status`, a compatible KMS plugin should return `v2` as `StatusResponse.version`,
   "ok" as `StatusResponse.healthz` and a `key_id` (remote KMS KEK ID) as `StatusResponse.key_id`.
 
```

Comment:
Should we mention that there was a v2beta1 API? People might have a plugin that supports this API.

Also, are we announcing a deprecation period for that beta API (we probably should, unless it's deprecated-but-permanently-supported)?

Reply:
They are both exactly the same and both will be supported forever as there is no value in breaking plugins.  How should we document that?

Reply:
See https://github.com/kubernetes/website/pull/43398#pullrequestreview-1737781121 for how I'd write it.